### DISCUSSION TITLE
Include source IP of user who started job in logs

### DISCUSSION THREAD

**Discussion by dforbes-pivotal** [concourse/concourse#6456] [Ideas]
Concourse is currently being used to manage our Cloud Foundry environment and as such is the mechanism by which changes are deployed to the environment.  A recent security audit noticed that the log files do capture the logged in user but not the source IP they were using when starting the job.  Should a user account be compromised the audit trail needs to be able to capture the source IP of the workstation used in the hope that this would allow the incident to be fully investigated.

Is it possible to ensure that the webserver log of Concourse (for Bosh) includes the source IP for all log entries captured?

**Comment by taylorsilva** [comment]
Hey, this was a really easy change. It's in this PR: https://github.com/concourse/concourse/pull/6462

**Reply by taylorsilva** [reply]
PR not merged: https://github.com/concourse/concourse/pull/6462#issuecomment-786280589

**Reply by taylorsilva** [reply]
Opened an issue for this: https://github.com/concourse/concourse/issues/6604
